README.md

```diff
@@ -76,29 +76,45 @@ To run the tests locally, you need the following:
 * You can disable the integration and/or junit tests via the following properties: `-Dintegration="no" -Djunit="no"`
 * If you do not define the `port_get`, `port_post`, and/or `port_data` properties, the respective test classes will not be executed
 
-## Examples
+## Documentation
+
+[Install Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html) before running the following commands e.g. by using this commands:
+```bash
+cd docs
+pip3 install -r requirements.txt
+```
+
+To update the ohsome API swagger files for the documentation:
+```bash
+cd docs
+wget 'https://api.ohsome.org/v1/docs?group=Data%20Aggregation' -O _static/swagger-aggregation.json
+wget 'https://api.ohsome.org/v1/docs?group=Data%20Extraction' -O _static/swagger-extraction.json
+wget 'https://api.ohsome.org/v1/docs?group=Metadata' -O _static/swagger-metadata.json
+```
 
-This section gives you an overview on analysis and services, that were/are using the ohsome API, as well as a JSON response example.<p>
-<p>
+The documentation can be built with the following command:
+```bash
+cd docs
+make clean # if you want to recreate all pages
+make html
+```
+
+If you want to see the release version of the ohsome API docs, use this environment variable:
+```bash
+cd docs
+DOCS_DEPLOYMENT=release make clean html
+```
 
+## Examples
 
-<p>
-The following blog posts describe analysis, which were using the ohsome API:
+The following blog posts describe analysis which were using the ohsome API:
    
 * [Farm shops are ohsome](http://k1z.blog.uni-heidelberg.de/2019/07/05/farm-shops-are-ohsome/)
 * [Plausible Parrots - HeiGIT’s OSHDB Supports Research in Citizen Science Data Quality](http://k1z.blog.uni-heidelberg.de/2019/02/27/plausible-parrots-heigits-oshdb-supports-research-in-citizen-science-data-quality/)
```

Comment:
I would drop this example, since it didn't acutally use the ohsome API, but an early version of the OSHDB.